src/test/java/org/wmn4j/notation/elements/ChordTest.java

```diff
@@ -126,30 +127,25 @@ public void testContainstNote() {
 
 	@Test
 	public void testGetDuration() {
-		assertFalse(this.cMajor.getDuration().equals(Durations.SIXTEENTH));
-		assertTrue(this.dMajor.getDuration().equals(Durations.QUARTER));
+		assertNotEquals(Durations.SIXTEENTH, this.cMajor.getDuration());
+		assertEquals(Durations.QUARTER, this.dMajor.getDuration());
 	}
 
 	@Test
 	public void testGetNote() {
-		assertTrue(this.cMajor.getNote(1).equals(Note.of(Pitch.of(Pitch.Base.E, 0, 4), Durations.QUARTER)));
-		assertTrue(this.dMajor.getNote(2).equals(Note.of(Pitch.of(Pitch.Base.A, 0, 3), Durations.QUARTER)));
+		assertEquals(Note.of(Pitch.of(Pitch.Base.A, 0, 3), Durations.QUARTER), this.dMajor.getNote(2));
```

Comment:
Were you supposed to skip the assert with C major note and test only the one with a D major note?

Reply:
That was left out by mistake, it is fixed now.

Reply:
Great :+1: